flow/packet-headers/rsvp.yaml

```diff
@@ -0,0 +1,1222 @@
+            description: >-
+              A 32-bit unicast, host address.  Any network-reachable interface address is allowed here.
+              Illegal addresses, such as certain loopback addresses, SHOULD NOT be used.
+            format: ipv4
+            default: 0.0.0.0
+            features: [count]
+          x-field-uid: 2
+        prefix_length:
+          x-field-pattern:
+            description: >-
+              Prefix-length of ipv4 address.
+            format: integer
+            length: 8
+            default: 32
+            features: [count]
+          x-field-uid: 3
+        flags:
+          description: >-
+              0x01  local_protection_available, 0x02  local_protection_in_use
+          $ref: '#/components/schemas/Flow.RSVPRecordRouteIPv4.Flag'
+          x-field-uid: 4
+    Flow.RSVPRecordRouteIPv4.Flag:
+      type: object
+      properties:
+        choice:
+            type: string
+            default: local_protection_available
+            x-enum:
+              local_protection_available:
+                x-field-uid: 1
+              local_protection_in_use:
+                x-field-uid: 2
+            x-field-uid: 1   
+    Flow.RSVP.PathRecordRouteType1Label:
+      description: >-
+        Class = RECORD_ROUTE, Type1 ROUTE_RECORD C-Type = 1
+        Subobject: Label, C-Type: 3
+      type: object
+      properties:
+        length:
+          description: >-
+              The Length contains the total length of the subobject in bytes, including the Type and Length fields.  
+              The Length MUST be atleast 4, and MUST be a multiple of 4.
+          $ref: '#/components/schemas/Flow.RSVPRouteRecord.Length'
+          x-field-uid: 1
+        flags:
+          x-field-pattern:
+            description: >-
+              0x01 = Global label. This flag indicates that the label will be understood if received on any interface.
+            format: integer
+            length: 8
+            default: 1
+          x-field-uid: 2 
+        c_type:
+          x-field-pattern:
+            description: >-
+              The C-Type of the included Label Object. Copied from the Label object.
+            format: integer
+            length: 8
+            default: 0
+          x-field-uid: 3
+        contents_of_label_obejct:
+          description: >-
+            The contents of the Label Object. Copied from the Label Object.
+          type: string
+          format: hex
+          default: "00"
+          x-field-uid: 4
+    Flow.RSVPRouteRecord.Length:
+      type: object
+      properties:
+        choice:
+          description: auto or configured value.
+          type: string
+          default: auto
+          x-field-uid: 1
+          x-enum:
+            auto:
+              x-field-uid: 1
+            value:
+              x-field-uid: 2
+        auto:
+          description: >-
+            The OTG implementation can provide a system generated value for this property.
```

Comment:
please check this description, the changes might be similar to the ones suggested earlier

Reply:
Done.